pkg/runtime/retriableError.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runtime
+
+import (
+	"fmt"
+)
+
+type RetriableError struct {
```

Comment:
I'm not sure if `retriable` is typo, that's how it was written before and I've followed the previous way of writing it

Reply:
Maybe it should be retryable?

Reply:
https://english.stackexchange.com/questions/305273/retriable-or-retryable

Retriable works :)